_posts/2024-01-16-Dhaka.md

```diff
@@ -0,0 +1,24 @@
+---
+title: دانشگاه داکا 
+tags: [آموزش زبان فارسی حضوری, آموزش زبان فارسی الکترونیکی همزمان, تالیف کتاب, برگزاری آزمون تعیین سطح]
+date: 2024-01-16 10:06:00 +0330
+image: 
+    path: /assets/img/logos/University of Dhaka.png
+    alt: دانشگاه داکا
+website: https://du.ac.bd/body/PERS
+email: 	persian@du.ac.bd
+address: 3017-خا، گروه زبان و ادبیات فارسی، طبقه 2، ساختمان هنر، دانشگاه داکا
+3017-Kha, Department of Persian Language and Literature, 2nd Floor, Arts Building, University of 3017-Dhaka
```

Comment:
The address should be in one line. Move the Persian or English address to the bottom of the page.